src/Nordea/Components/Slider.elm

```diff
@@ -0,0 +1,139 @@
+                , NordeaText.textTinyLight
+                    |> NordeaText.view [ css [ color Colors.grayNordea ] ] [ Html.text config.description ]
+                ]
+            , NumberInput.init config.value
+                |> NumberInput.withMin config.min
+                |> NumberInput.withMax config.max
+                |> NumberInput.withStep (config.step |> Maybe.withDefault 1)
+                |> NumberInput.withOnInput config.onInput
+                |> NumberInput.withError ((config.value |> String.toFloat |> Maybe.withDefault 1) > config.max || (config.value |> String.toFloat |> Maybe.withDefault 1) < config.min)
+                |> NumberInput.view [ name "rangeInput", css [ flex (int 1) ] ]
+            ]
+        , input
+            [ name "rangeInput"
+            , type_ "range"
+            , Html.Styled.Attributes.value config.value
+            , Html.Styled.Attributes.min (config.min |> String.fromFloat)
+            , Html.Styled.Attributes.max (config.max |> String.fromFloat)
+            , Html.Styled.Attributes.step (config.step |> Maybe.map String.fromFloat |> Maybe.withDefault "1")
+            , config.onInput |> onInput
+            , css [ sliderStyle (Slider config) ]
+            ]
+            []
+        ]
+
+
+sliderStyle : Slider msg -> Style
+sliderStyle (Slider config) =
+    Css.batch
+        [ width (pct 100)
+        , property "-webkit-appearance" "none"
+        , cursor pointer
+        , margin2 (px 8) (px 0)
+
+        -- Webkit
+        , pseudoElement "-webkit-slider-runnable-track"
+            [ width (pct 100)
+            , height (px 4)
+            , property "background-color" "transparent"
+            ]
+        , pseudoElement "-webkit-slider-thumb"
+            [ property "-webkit-appearance" "none"
+            , width (px 20)
+            , height (px 20)
+            , borderRadius (pct 100)
+            , backgroundColor (hex "#00005E")
+            , marginTop (px -8.5)
+            ]
+
+        -- Mozilla
+        , pseudoElement "-moz-range-track"
+            [ width (pct 100)
+            , height (px 4)
+            , backgroundColor transparent
+            ]
+        , pseudoElement "-moz-range-thumb"
+            [ width (px 20)
+            , height (px 20)
+            , borderRadius (pct 100)
+            , backgroundColor (hex "#00005E")
+            , marginTop (px -8.5)
+            ]
+        , adjustSlider (Slider config)
+        ]
+
+
+adjustSlider : Slider msg -> Style
+adjustSlider (Slider config) =
+    let
+        visibleWidth =
+            ((((config.value |> String.toFloat |> Maybe.withDefault 0) - config.min) * 100) / (config.max - config.min)) |> String.fromFloat
+
+        cloudBlue =
+            "#DCEDFF"
+
+        nordeaBlue =
+            "#0000A0"
+
+        gradientValue =
+            "linear-gradient(to right," ++ nordeaBlue ++ " 0% " ++ visibleWidth ++ "%, " ++ cloudBlue ++ " " ++ visibleWidth ++ "%" ++ " 100% )"
```

Comment:
Can we use the colors from Colors.elm here instead of hardcoded strings? We used to have a `colorToString` function, so that may be used to convert them to strings if needed

Reply:
Nice, we'll use that!😁

Reply:
And also, support for themes would be nice 😀